### ISSUE TITLE
Add "external links" off switch

### ISSUE THREAD

**Issue by Popolechien** [kiwix/kiwix-hotspot#558]
From a user: _"I noticed that clicking links that go out to external pages with an "https" in the URL delivers an error message that's a bit disconcerting to non-technical users."_
<img width="508" alt="2021-01-23 23_11_42-Privacy error" src="https://user-images.githubusercontent.com/21675885/105832911-c8174c00-5fc8-11eb-94e9-19cd7f8b5b83.png">

Since most use cases relate to people being offline, there is no point in letting them click on links to nowhere that will worry them more than anything. Could it be possible to have a switch feature before building the image that inactivates these external links by default (but can be reactivated if need be)?

**Comment by rgaudin** [comment]
For kiwix-serve, we have a feature (not enabled on hotspot) that catches external links and displays a warning. We could enable that on the hotspot and let them know that if they get the warning, it's because the hotspot is not connected to Internet and this the external link is useless.

A second iteration could check the Internet connection status and behave differently (either different message or just redirect to the destination).

**Comment by kelson42** [comment]
@rgaudin Yes this is probably a bettre approach to configure this per default as the use case where a Kiwix-Hotspot is online is a rare one.

**Comment by HassanRahim26** [comment]
Hi, I am Hassan. I would like to work on this issue, can you suggest ways to get started?

**Comment by SRDdev** [comment]
Can I work on this issue! 
@Popolechien

**Comment by kelson42** [comment]
@SRDdev Yes, would you be able first to explain here how your solution woukd look like?

**Comment by SRDdev** [comment]
Sorry I thought about it and found some technical issues in my solution, you can pass it on @kelson42 
Thank you so much @kelson42 !!

**Comment by Popolechien** [comment]
@SRDdev if you found the issues, then you are halfway to solving them ;-)

**Comment by kelson42** [comment]
Fixed in Kiwix-Hotspot 3.0
